docs/src/configuration.md

```diff
@@ -179,3 +179,10 @@ Whenever to use a more intrusive, faster unwinding algorithm; enabled by default
 
 Setting it to `0` will on average significantly slow down unwinding. This option
 is provided only for debugging purposes.
+
+### `MEMORY_PROFILER_TRACK_CHILD_PROCESSES`
+
+*Default: `0`*
+
+If set to `1`, bytehound will also track the memory allocations of child processes spawned by the
+profiled process.
```

Comment:
Could you explicitly also document that:

1) this only applies to child processes which are `fork`d and `exec`d,
2) that the `MEMORY_PROFILER_OUTPUT` needs to be set with the appropriate placeholders so that the filenames of both the parent and the child are different

Reply:
Done.